Review styles reference (#1408)
Textualize/textual: docs/snippets/type_syntax/color.md

> The legal values for a [`<color>`](/css_types/color) depend on the [class `Color`][textual.color.Color] and include:

Reviewer: I don't think we need the reference to the Color class here.

```suggestion
A [`<color>`](/css_types/color) should be in one of the following formats:
```

Author: Fixed in 1ef20eb.

Is it worth adding a formal syntax description of the values accepted, much like we have for the styles?
Something like

```
color: named-color
     | hex-rgb
     | functional-rgb
     | functional-hsl

hex-rgb: # hex hex hex
       | # hex hex hex hex
       | # hex hex hex hex hex hex
       | # hex hex hex hex hex hex hex hex
...
```

 - a recognised [named color](../../api/color#textual.color--named-colors) (e.g., `red`);
 - a hexadecimal number representing the RGB values of the color (e.g., `#F35573`);
 - a color description in the RGB system (e.g., `rgb(23,78,200)`);
 - a color description in the HSL system (e.g., `hsl(290,70%,80%)`); and

> For more details about the exact formats accepted, see [the class method `Color.parse`][textual.color.Color.parse].

Reviewer: I think this page should be exhaustive regarding the color formats. Since we're talking about CSS, I don't think we should direct people to the code reference.

Author: Fixed in 1ef20eb.
